tool/cli.js

```diff
@@ -223,10 +223,10 @@ program
       }
       const { doc } = await buildDocument(document);
 
-      if (doc.flaws) {
-        const flaws = Object.values(doc.flaws)
-          .map((a) => a.length || 0)
-          .reduce((a, b) => a + b);
+      const flaws = Object.values(doc.flaws || {})
+        .map((a) => a.length || 0)
```

Comment:
I think I'm guilty of starting this. In the early early days of the design of flaws, I wasn't really certain how the structure should be. But now I think every flaw should always be an array. An array of objects. E.g. 
```
flaws == {
  some_key: [
     {key: value, key2: value},
     ...
  ]
}
```
and if anything doesn't conform to that we should kill it hard.